package.json

```diff
@@ -48,8 +48,8 @@
     "pre-commit:lint": "yarn lint-staged",
     "pre-commit:types": "yarn tsc",
     "pre-commit:test": "yarn test",
-    "test": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
+    "test": "cross-env CI=1 react-scripts test --env=jest-environment-jsdom-sixteen",
```

Comment:
This was necessary for some tests to work with hover / focus.

Reply:
Nerp, after I fixed some other stuff, I didn't need jsdom 16 after all. And including it broke backwards compatibility with Node 10 (which we say we support).
So I removed it, but I'm kinda sad that Node 10 support might stop us from upgrading stuff, in general.